README.rst

```diff
@@ -23,6 +21,37 @@ machine code.
 For more information, see `the documentation
 <https://cretonne.readthedocs.io/en/latest/?badge=latest>`_.
 
+Status
+------
+
+Cretonne currently supports enough functionality to run a wide variety of
+programs, including all the functionality needed to execute WebAssembly MVP
+functions, although it needs to be used within an external WebAssembly
+embedding to be part of a complete WebAssembly implementation.
+
+The x86-64 backend is currently the most complete and stable; other
+architectures are in various stages of development. Cretonne currently supports
+the System V AMD64 ABI calling convention used on many platforms, but does not
+yet support the Windows x64 calling convention. The performance of code
+produced by Cretonne is not yet impressive, though we have plans to fix that.
+
+The core codegen creates have minimal dependencies, and do not require any host
```

Comment:
`The core codegen creates have minimal dependencies, and do not require` Is the grammar correct here? I'm having trouble understanding this sentence.

Reply:
I think there is a typo: creates → crates

Reply:
@bjfish If I split it into two sentences, does that clarify it? "The core codegen creates have minimal dependencies. And, they do not require any host floating-point support."

Reply:
@sunfishcode I still don't understand. Is there a typo? Should it be: "The *code* codegen creates *has* minimal dependencies." or "The core codegen *crates* have minimal dependencies." Or is "creates" actually a noun in this context? If so what are "creates"?

Reply:
Ah, I see it now. It's a typo. "creates" should be "crates". Thanks for spotting that!